test/svg-test-helper.js

```diff
@@ -181,3 +250,12 @@ export const isCircle = (commandString) =>
  */
 export const isTriangle = (commandString) =>
   exhibitsShapeSequence(commandString, TRIANGULAR_SEQUENCE);
+
+/**
+ * Determines if a circular sector (slice of pie) shape is produced from the provided path command.
+ *
+ * @param {String} commandString - The command attribute of a `path` element.
+ * @returns {Boolean}            - Boolean indicating if the command string produces a circular sector shape.
+ */
+export const isCurcularSector = (commandString) =>
```

Comment:
Good catch. It would be really cool if there was like a VS Code extension or something where you could configure your casing pattern and it would run a spellcheck against a dictionary.

Reply:
WebStorm for the win :)